packages/superinput/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@botpress/superinput",
+  "version": "0.1.0",
+  "private": true,
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "@codemirror/autocomplete": "^0.19.9",
+    "@codemirror/closebrackets": "^0.19.0",
+    "@codemirror/commands": "^0.19.5",
+    "@codemirror/highlight": "^0.19.6",
+    "@codemirror/history": "^0.19.0",
+    "@codemirror/lang-javascript": "^0.19.3",
+    "@codemirror/language": "^0.19.6",
+    "@codemirror/rangeset": "^0.19.2",
+    "@codemirror/state": "^0.19.6",
+    "@codemirror/text": "^0.19.5",
+    "@codemirror/tooltip": "^0.19.10",
+    "@codemirror/view": "^0.19.21",
+    "@lezer/common": "^0.15.9",
+    "@lezer/markdown": "^0.15.2",
+    "espree": "^9.3.0",
+    "fast-memoize": "^2.5.2",
+    "react-json-view": "^1.21.3",
+    "tree-model": "^1.0.7",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "build": "yarn doctree && rollup -c",
+    "dev": "rollup -c -w",
+    "demo": "parcel demo/index.html",
+    "doctree": "ts-node --skip-project ./scripts/genDocTree.ts"
+  },
```

Comment:
Something weird I noticed is that the SuperInput package has two dev dependencies for two different bundlers. It uses parcel for the demo, but it uses rollup to produce the lib. Parcel could also produce a bundle using parcel build. I think it would be best to avoid extra dependencies when possible.

If we consider that the ui-studio is using webpack to bundle itself as well, we have a total of 3 bundlers in this repo. I'm thinking of adjusting the studio so it can use parcel instead. We also use babel in the studio which I'm thinking of removing as well.

So ideally we would be using tsc for the backend, with ts-node for development, and parcel for the frontend to bundle and for development. We should also make sure that the code in the ui packages can be compiled using tsc from time to time, since tsc is often stronger for type checking than bundlers like rollup and parcel. It can also spot issue in the project configuration (like the tsconfig), that bunlders sometime ignore.

@EFF @laurentlp @rob-gordon @Raid55 what do you think.

This is not necessary to resolve to merge this PR, but it's something to thing about for the repo as a whole.